### DISCUSSION TITLE
Welcome to ISIS3 Discussions!

### DISCUSSION THREAD

**Discussion by jlaura** [DOI-USGS/ISIS3#4654] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michaelaye** [comment]
Is this in beta and meant to replace the discourse forum in the future?

**Comment by Sagar1711** [comment]
How do I install ISIS3 using zip files from github release assets?

**Reply by jessemapel** [reply]
The github releases are just the source code. You'll need to build and install it from the release assets. If you want to just install pre-compiled binaries, use Anaconda.

**Reply by Sagar1711** [reply]
How do I build and install it from release assets?
Thanks

**Reply by jessemapel** [reply]
Follow these instructions. It requires Anaconda https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-isis3

**Reply by Sagar1711** [reply]
Hi,
I followed the instructions given in the link, to build and install ISIS3 from release assets. But I have been getting following error at ninja install or ninja -j 8 install commands
`In file included from /home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/random:51,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/BundleResultsTests.cpp:3:
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:2714:5: note: candidate: 'template<class _IntType, class _CharT, class _Traits> std::basic_ostream<_CharT, _Traits>& std::operator<<(std::basic_ostream<_CharT, _Traits>&, const std::discrete_distribution<_IntType>&)'
 2714 |     operator<<(std::basic_ostream<_CharT, _Traits>& __os,
      |     ^~~~~~~~
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:2714:5: note:   template argument deduction/substitution failed:
In file included from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-matchers.h:48,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/internal/gtest-death-test-internal.h:46,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-death-test.h:43,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest.h:60,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/Fixtures.h:4,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/BundleResultsTests.cpp:25:
/home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-printers.h:212:33: note:   'const QChar' is not derived from 'const std::discrete_distribution<_IntType>'
  211 |             typename = decltype(std::declval<std::ostream&>()
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  212 |                                 << std::declval<const T&>())>
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/random:51,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/BundleResultsTests.cpp:3:
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:2936:5: note: candidate: 'template<class _RealType, class _CharT, class _Traits> std::basic_ostream<_CharT, _Traits>& std::operator<<(std::basic_ostream<_CharT, _Traits>&, const std::piecewise_constant_distribution<_RealType>&)'
 2936 |     operator<<(std::basic_ostream<_CharT, _Traits>& __os,
      |     ^~~~~~~~
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:2936:5: note:   template argument deduction/substitution failed:
/home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-printers.h:212:33: note:   'const QChar' is not derived from 'const std::piecewise_constant_distribution<_RealType>'
  211 |             typename = decltype(std::declval<std::ostream&>()
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  212 |                                 << std::declval<const T&>())>
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/random:51,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/BundleResultsTests.cpp:3:
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:3142:5: note: candidate: 'template<class _RealType, class _CharT, class _Traits> std::basic_ostream<_CharT, _Traits>& std::operator<<(std::basic_ostream<_CharT, _Traits>&, const std::piecewise_linear_distribution<_RealType>&)'
 3142 |     operator<<(std::basic_ostream<_CharT, _Traits>& __os,
      |     ^~~~~~~~
/home/sshar200/mambaforge/envs/isis/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/random.tcc:3142:5: note:   template argument deduction/substitution failed:
In file included from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-matchers.h:48,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/internal/gtest-death-test-internal.h:46,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-death-test.h:43,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest.h:60,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/Fixtures.h:4,
                 from /home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/isis/tests/BundleResultsTests.cpp:25:
/home/sshar200/env-setup-test/ISIS3-7.1.0_RC1/gtest/googletest/include/gtest/gtest-printers.h:212:33: note:   'const QChar' is not derived from 'const std::piecewise_linear_distribution<_RealType>'
  211 |             typename = decltype(std::declval<std::ostream&>()
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  212 |                                 << std::declval<const T&>())>
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.`

**Reply by jessemapel** [reply]
What version of gtest are using? Try building with tests disabled by passing `-DbuildTests=OFF` to cmake in the configuration step.
